diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/EmptyBlock.kt

```diff
@@ -41,7 +36,10 @@ class EmptyBlock(configRules: List<RulesConfig>) : DiktatRule(
         checkEmptyBlock(newNode, configuration)
     }
 
-    @Suppress("UnsafeCallOnNullableType")
+    private fun isNewLine(node: ASTNode) =
+            node.findChildByType(WHITE_SPACE)?.text?.contains("\n") ?: true
```

Comment:
Why is it `true` when white space node is missing?

Reply:
Yes, there must be a false